guides/v2.1/contributor-guide/contributing.md

```diff
@@ -12,70 +12,71 @@ redirect_from: /guides/v1.0/contributor-guide/contributing.html
 
-Often when the Community Engineering Team works on reviewing the suggested changes, we will add a label to the issue to indicate certain information, like the status or who is working the issue. If you're ever curious what the different labels mean, see the <a href="#labels">table</a> below for an explanation of each one.
+Often when the Community Engineering Team works on reviewing the suggested changes, we will add a label to the issue to indicate certain information, like the status or who is working the issue. If you're ever curious what the different labels mean, see the [table](#labels) below for an explanation of each one.
 
 <div class="bs-callout bs-callout-info" id="info">
 <p>Please refer to <a href="http://www.magento.com/legaldocuments/mca">Magento Contributor Agreement</a> for detailed information about the License Agreement. All contributors are required to submit a click-through form to agree to the terms. </p>
 </div>
 
-<h2 id="question">Questions or enhancement requests?</h2>
+## Questions or enhancement requests? {#question}
+
+We use this repository (the Magento 2 GitHub repository) to capture code and documentation issues. We recommend that you post all questions to a question-and-answer site, such as [Stack Exchange](https://magento.stackexchange.com/){:target="\_blank"} and the [Magento Forums](https://community.magento.com/){:target="\_blank"}, where Magento community members can quickly provide recommendations and advice.
 
-We use this repository (the Magento 2 GitHub repository) to capture code and documentation issues. We recommend that you post all questions to a question-and-answer site, such as <a href="https://magento.stackexchange.com/" target="_blank">Stack Exchange </a> and the <a href="https://community.magento.com/" target="_blank">Magento Forums</a>, where Magento community members can quickly provide recommendations and advice.
+Submit feature requests or enhancement suggestions to the new [Magento 2 Feature Requests and Improvements forum](https://community.magento.com/t5/Magento-2-Feature-Requests-and/idb-p/feature-requests){:target="\_blank"} (see details [here](https://community.magento.com/t5/News-Announcements/Improvements-to-GitHub-Management/m-p/44572#M96){:target="\_blank"}).
 
-Submit feature requests or enhancement suggestions to the new <a href="https://community.magento.com/t5/Magento-2-Feature-Requests-and/idb-p/feature-requests" target="_blank">Magento 2 Feature Requests and Improvements forum</a> (see details <a href="https://community.magento.com/t5/News-Announcements/Improvements-to-GitHub-Management/m-p/44572#M96" target="_blank">here</a>).
+## Accepted pull requests and ported code {#rules}
 
+Submit pull requests and ported code according to the following rules:
 
```

Comment:
It would be better to replace next list with table:

| | Fix for Existing Issue | Test Coverage | Refactoring| New Feature | Code Cleanup
| --- | :---: | :---: | :---: | :---: | :---: | 
2.1 | ✅ | ✅ | ❌ | ❌ | ❌
2.2 | ✅ | ✅ | ✅ | ❌ | ❌
2.3 | ✅ | ✅ | ✅ | ✅ | ✅

Reply:
@sidolov Revised and added the table and an opening sentence or two describing the rules. Rebuilding to verify content generates correctly. Will slack a link when done for final approval. Thanks!